web/public/js/services/cd-users-service.js

```diff
@@ -42,6 +42,13 @@ function cdUsersService(cdApi, $q){
         cdApi.post('profiles/youth/create', {profile: profile}, win, fail || topfail);
       }
     },
+    saveYouthProfilePromise: function(profile) {
```

Comment:
Fair point. Just have to add returns to the original message and cd-api will handle it for us.